core/commons/che-core-commons-test/pom.xml

```diff
@@ -21,10 +21,30 @@
     <artifactId>che-core-commons-test</artifactId>
     <packaging>jar</packaging>
     <name>Che Core :: Commons :: Utilities for tests</name>
+    <properties>
+        <findbugs.failonerror>true</findbugs.failonerror>
+    </properties>
     <dependencies>
         <dependency>
+            <groupId>com.google.inject</groupId>
+            <artifactId>guice</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>javax.inject</groupId>
+            <artifactId>javax.inject</artifactId>
```

Comment:
Can you please explain why this dependency should be provided?
Guava is provided because dependency analyzer will fail with an error 
```
[WARNING] Unused declared dependencies found:
[WARNING]    com.google.guava:guava:jar:18.0:compile
```

But we can't remove, beucase we will get the following error:
```
[ERROR] java.lang.NoClassDefFoundError: com/google/common/collect/Sets
```